### DISCUSSION TITLE
ETA for the Notice Editor

### DISCUSSION THREAD

**Discussion by chkuck** [OP-TED/eForms-SDK#109] [Q&A] [answered]
Do you have a rough estimate of when the sample "Notice Editor" application mentioned in the roadmap will be available?

With many of the problems we have with development, it would be helpful to be able to view an existing solution.

**Comment by rousso** [comment]
Hi @chkuck,

I am glad to see you are interested in this. 
To be honest we have been absorbed by our SDK deadlines and we have left this aside for a while. 

Your interest however motivates me to try to expedite this.  
Let me check where we are and how fast we can get it in your hands and I will get back to you.

Regards,
Yannis

cc: @rouschr, @meletev

**Comment by chkuck** [comment, marked as answer]
Thank you very much for making https://github.com/OP-TED/eforms-notice-editor public :)
We will review the code and see if it help with our problems.

Best regards,
Christian

**Comment by rousso** [comment]
Thank you @chkuck. 

We are working on the notice editor to provide a sample implementation for generating the XML file.
Please do not hesitate to give us your feedback. 
You can always monitor our commits on the develop branch to get our the latest unreleased code.

I will mark this question as answered as it was about the ETA.

Regards,
Yannis

**Comment by rouschr** [comment]
The next features on the Editor roadmap are:
1. Saving a notice to XML (see discussions here https://github.com/OP-TED/eForms-SDK/discussions/126)
2. Loading an existing notice
